client/components/splash_codes/_edit.html

```diff
@@ -8,7 +8,7 @@
   <md-divider></md-divider>
 </md-toolbar>
 
-<md-content class="md-padding" layout="column" layout-gt-sm='row'>
+<md-content class="md-padding" layout="column" layout-gt-sm='column'>
```

Comment:
@simonmorley , yes, it was; double checked on my phone - looks the same as it does before that